product_docs/docs/edbcloud/beta/overview/03_security_compliance.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: "Security and Compliance"
+---
+
+### Security Governance
+
+- Cloud Security Compliance officer is responsible for maintaining compliance targets for their entire organization.
+- EDB Cloud infrastructure periodically undergoes SOC2 audit trials to ensure that EDB Cloud is always compliant with security requirements.
+- All EDB staff undergo background checks before they are hired.
+- All EDB staff are required to take mandatory security training.
+- The EDB Software Development Life Cycle integrates continuous security scans of added/updated code.
+
+
+### EDB Cloud Security Features
+EDB Cloud is a Database as a Service (DBaaS) which runs in your own Cloud account, isolating your data from other users & giving you control over our access to it. The key security features of EDB Cloud are as follows:
+- **Segregation of IT environments:** Customers database clusters are installed and managed in their own cloud environment. Complete segregation of customer's data is ensured.
+- **Access Control:** EDB Cloud supports _Single Sign on_ and _Role Based Access Control_ policies. Customers may define their own sets of _Roles_ and  _Role Based Access Control_ policies to manage their individual cloud environments.
+ 
+- **Data Encryption:** EDB Cloud encrypts customer data in motion and at rest. All network traffic is encrypted using Transport Layer Security (TLS). Data stored in database clusters is encrypted appropriately. Users can bring their own encryption keys for an additional level of control.
+- **Logging and Auditing:** EDB Cloud supports Logging and Auditing
+- **Backups:** EDB Cloud data is continuously replicated from the production to the hot standby data center.
+- **Disaster Recovery and Business Continuity:** All components of EDB Cloud infrastructure are highly available and redundant.
+- **Incident Response:** EDB employs tools and processes which monitor the platform and has a dedicated security team as well as incident response processes.
+
+### Shared Responsibility
```

Comment:
We should have some words here about not touching the Subscriptions we're working in, eh?